src/AppBundle/Resources/views/themes/app/content/full/ng_article.html.twig

```diff
@@ -84,7 +84,7 @@
                 },
             {% endif %}
             {% if not content.fields.image.empty %}
-                "image": "{{ absolute_url(ng_image_alias(content.fields.image, 'i1200').uri) }}",
+                "image": "{{ ngsite_image_absolute_url(content, 'image', 'i1200') }}",
```

Comment:
Together with my comment from `netgen/site-bundle` repo, this would then become `absolute_url(ngsite_image_url(content, 'image', 'i1200'))`, right?

Reply:
Yes, I thought of that, and I see it as we're just making sure that the url is absolute. Same as e.g. making sure that the string is lowercased, when we're not sure if it is.

Reply:
Changed in 10a0c235c66c15b0fa5162c7cdac24bf27ecd2a6